pkg/services/secrets/kvstore/kvstore.go

```diff
@@ -14,29 +15,31 @@ const (
 )
 
 func ProvideService(sqlStore sqlstore.Store, secretsService secrets.Service, remoteCheck UseRemoteSecretsPluginCheck) SecretsKVStore {
+	var store SecretsKVStore
 	logger := log.New("secrets.kvstore")
+	store = &secretsKVStoreSQL{
```

Comment:
I thought it would be better to have it on top since its the fallback implementation, this way we don't need to call `NewCachedKVStore(store, 5*time.Second, 5*time.Minute)` for each case, default, vaid plugin, and invalid plugin.

Reply:
Oh yeah, totally missed that. Sounds good!